src/detect-engine-threshold.c

```diff
@@ -821,7 +823,16 @@ static int ThresholdCheckUpdate(const DetectThresholdData *td, ThresholdEntry *t
                     te->current_count = 1;
                 }
             }
+            if (de_ctx->RateFilterCallback && original_action != pa->action) {
+                pa->action = (*de_ctx->RateFilterCallback)(p, sid, gid, rev, original_action,
```

Comment:
can this just be `de_ctx->RateFilterCallback(p, ...)`?

Reply:
Yup.